flux-cli/src/main/java/com/marklogic/flux/api/ClassifierOptions.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2025 MarkLogic Corporation. All Rights Reserved.
+ */
+package com.marklogic.flux.api;
+
+/**
+ * @since 1.2.0
+ */
+public interface ClassifierOptions {
+
+    ClassifierOptions classifierHost(String host);
```

Comment:
No need for the "classifier" prefix here, it's already implied by the class name. 

"since" annotation should be 1.3.0.

But - like mentioned before, I generally wait on adding the API stuff until everything is working. I only do the CLI options and I test everything with the CLI options. Once it's all stable, then I toss on the API options and add some smoke tests for those. I think it's a lot less work that way.

Reply:
I made the changes, but left this stuff in. Let me know if you want me to pull it out.